### ISSUE TITLE
Destinations star - pass through -  end

### ISSUE THREAD

**Issue by paaljoachim** [WordImpress/google-maps-builder#92]
Destinations

Should there be an obvious start position and an end position? Clearly marking this.
1. option. Have a Start above the location text field. Have an End above the location text field of the second destination. Have a pass through this location over the location text field of all destinations between start and end.

2. option 
A checkbox under each with the options: start, pass through, end.

**Comment by mathetos** [comment]
Considering you can customize your marker yourself I'm not sure this is necessary.

@DevinWalker  Thoughts?

**Comment by DevinWalker** [comment]
I'm not sure about this request either. The destinations should be evident as it's handled via the Google Maps API. Until further user input, I'll consider it good to go for now.

**Comment by paaljoachim** [comment]
I noticed the title said destination star. It should have been start. My mistake.
One can have multiple locations/destinations list. One assumes the start destination is at the top, various locations one want to pass through below and the last destination is then the end destination.
Here I am visually showing what I mean. I also added triangles on the left showing that one can drag a destination around. 

![start-to-end-destination](https://cloud.githubusercontent.com/assets/5323259/9427315/3bee25f4-4977-11e5-8b8a-704ce2c8fe6b.jpg)
